packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -41,7 +41,10 @@ export function getPropGetters<
       // Right now, a second instance makes this computation return false.
       onTouchStart(event) {
         if (
```

Comment:
I'm not sure to understand exactly what line 47 does. Could extracting the first condition to a variable help?

I think we can remove (or rephrase) the comment because it only applies to `onInput.isRunning()` branches.

A suggestion, if that naming makes sense:

```js
        const shouldSkipAction =
          store.getState().isOpen === false && !onInput.isRunning();

        if (shouldSkipAction || event.target === inputElement) {
          return;
        }
```

Reply:
We're using `onTouchStart` to handle `blur` on mobile instead of ther native `blur` event (cf. your comment l. 35). But `onTouchStart` triggers in many more occasions than `blur` so we have the branch in l. 43 to return when this isn't a blur scenario.

So far, whenever the panel was closed or the touched element was the input, we returned: these cases indicate we're not in a blur scenario. But we could be: if you have `openOnFocus=false` (therefore, `isOpen=false` until there are results), you type a character and the query takes a while, then you tap outside, you'd fall into the branch, and return. But you actually want the `blur` logic to run.

I'll extract this into a more explicit variable and comment so we don't lose this context.

Reply:
Addressed in 43107e5.